services/language_translation/v2.js

```diff
@@ -21,358 +22,353 @@ module.exports = function (RED) {
 
   temp.track();
 
-  var username, password, language_translation;
+  // Require the Cloud Foundry Module to pull credentials from bound service 
+  // If they are found then they are stored in sUsername and sPassword, as the 
+  // service credentials. This separation from sUsername and username to allow 
+  // the end user to modify the node credentials when the service is not bound.
+  // Otherwise, once set username would never get reset, resulting in a frustrated
+  // user who, when he errenously enters bad credentials, can't figure out why
+  // the edited ones are not being taken.
+
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['services' is defined but never used.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2613212642/issues/source?bid=3225115&fileBranchId=3311770#l33)